### PR TITLE
Update 2016-ideas.post

### PR DESCRIPTION
**Pull request by Walkman100**
Removed some redundant words, added some commas and a period to improve readability.

[Review on Reviewable](https://reviewable.io/reviews/deavmi/blog/5)

### CODE REVIEW

File: 2016-ideas.post [hunk 1]
```diff
@@ -3,23 +3,23 @@ author="Tristan B. Kildaire"
...
+This project is aimed at creating a continuous build server that will build \`.acorn\` files continuously and spit out binaries.
 
 <h2>So what else?</h2>
 
-Well, I'd like to close off this blog post now but I can say one thing the two projects mentioned above go very well together and I think creating both will really be great and will create a world of simplistic systems where maintaining stuff is really easy and such. The **Acorn Build System** is no replacement for the **make** build system but rather a tool to bind many build tools and compilers together in one uniform syntax and tool with a side-project that can continuously build binaries from \`.acorn\` files.
+Well, I'd like to close off this blog post now but I can say one thing. The two projects mentioned above go very well together, and I think creating both will really be great and will create a world of simplistic systems where maintaining stuff is really easy. The **Acorn Build System** is no replacement for the **make** build system, but rather a tool to bind many build tools and compilers together in one uniform syntax, with a side-project that can continuously build binaries from \`.acorn\` files.
```
Comment (Walkman100) [inline 2016-ideas.post]: I think this should be `but I can say one thing: the two projects` rather, but I'm not sure

**Comment by deavmi** [comment]
Thank you.

**Comment by deavmi** [comment]
@Walkman100 or maybe instead of a colon you could use a comma?

**Comment by Walkman100** [comment]
the point of a PR is to discuss the changes before merging...

anyway, no, I changed it to a period, but I'm thinking a colon might be better. a comma would be worse.

**Pull request merged**